src/commands/link/link.ts

```diff
@@ -266,33 +249,33 @@ export const link = async (options: OptionValues, command: BaseCommand) => {
 
   // Site id is incorrect
   if (siteId && isEmpty(siteData)) {
-    log(`"${siteId}" was not found in your Netlify account.`)
-    log(`Please double check your siteID and which account you are logged into via \`netlify status\`.`)
-    return exit()
+    NetlifyLog.error(`"${siteId}" was not found in your Netlify account.`)
+    outro({
+      message: `Please double check your siteID and which account you are logged into via \`netlify status\`.`,
```

Comment:
Know this existed before but I think there needs to be a space here

```suggestion
      message: `Please double check your site ID and which account you are logged into via \`netlify status\`.`,
```